js/index.js

```diff
@@ -57,49 +60,38 @@ const searchRequest = search => {
 };
 
 const makeRequest = fullURL => {
-  httpRequest = new XMLHttpRequest();
-  if (!httpRequest) {
-    alert("Can't create the request");
-    return false;
-  }
-
-  httpRequest.onreadystatechange = function () {
-    if (this.readyState == 4 && this.status == 200) {
-      responseContent = httpRequest.responseText;
-      parsed = JSON.parse(responseContent);
+  httpRequest = new Request(fullURL, { method: 'GET' });
+
+  fetch(httpRequest)
+    .then(function (response) {
+      return response.json();
+    })
+    .then(function (data) {
```

Comment:
Pls use arrow functions

Reply:
Sure, don't know why I've forgotten. I just intended to test this piece of code with regular functions and then switch to arrow functions but I forgot about them :D

Reply:
Yeah, it happens... still not fully believe in ()  => {} 😃